fuzzers/001-part-yaml/generate.tcl

```diff
@@ -22,13 +22,17 @@ write_checkpoint -force design.dcp
 # Write a normal bitstream that will do a singe FDRI write of all the frames.
 write_bitstream -force design.bit
 
-# Write a debug bitstream which writes each frame individually followed by
-# the frame address.  This shows where there are gaps in the frame address
-# space.
-set_property BITSTREAM.GENERAL.DEBUGBITSTREAM YES [current_design]
-write_bitstream -force design.debug.bit
-set_property BITSTREAM.GENERAL.DEBUGBITSTREAM NO [current_design]
-
-set_property BITSTREAM.GENERAL.PERFRAMECRC YES [current_design]
-write_bitstream -force design.perframecrc.bit
-set_property BITSTREAM.GENERAL.PERFRAMECRC NO [current_design]
+if { ![info exists ::env(XRAY_XC7Z)] } {
```

Comment:
Does DEBUGBITSTREAM work for everything? Might be better to convert everything over instead of making this conditional

Reply:
DEBUGBITSTREAM does not work for xc7z.  However, PERFRAMECRC should work everywhere.

Reply:
Can you test PERFRAMECRC on all three? Lets drop DEBUGBITSTREAM if it works on everything

Reply:
PERFRAMECRC worked on all three and produced identical YAMLs as DEBUGBITSTREAM. I think we can switch to CRC

Reply:
the PR now changes the 001 fuzz to use PERFRAMECRC for all the devices